sky/serve/replica_managers.py

```diff
@@ -124,10 +126,14 @@ def launch_cluster(task_yaml_path: str,
 
 # TODO(tian): Combine this with
 # sky/spot/recovery_strategy.py::terminate_cluster
-def terminate_cluster(cluster_name: str, max_retry: int = 3) -> None:
+def terminate_cluster(cluster_name: str,
+                      max_retry: int = 3,
+                      delay_in_s: int = 0) -> None:
     """Terminate the sky serve replica cluster."""
+    time.sleep(delay_in_s)
```

Comment:
What do we set `delay_in_s` to? Do we need this in addition the the scale-down delay we already have?

Reply:
Ah this is for draining I'm guessing? To make sure ongoing requests are routed to new instance?

Reply:
You are correct. This is a experimental feature from the spot policy branch. And @MaoZiming mentioned that some more discussion needs to be done weather or not to keep this `time.sleep()`